test/helpers/cert_utils.go

```diff
@@ -44,14 +45,17 @@ func GenerateSelfSignedCert(t testing.TB) (keyBytes, certBytes []byte) {
 	keyBytes = x509.MarshalPKCS1PrivateKey(key)
 
 	tmpl := x509.Certificate{
-		NotBefore:    time.Now(),
+		NotBefore:    time.Now().Add(-10 * time.Minute), // Allow for clock skew
```

Comment:
Just curious why 10 minutes needs to be removed from the time

Reply:
just incase there is clock drift

Reply:
I removed it